compiler+runtime/src/cpp/jank/runtime/obj/chunked_cons.cpp

```diff
@@ -167,7 +167,7 @@ namespace jank::runtime::obj
             return false;
           }
         }
-        return true;
+        return seq == nullptr;
```

Comment:
I don't think this is correct. In the case where `this` is a shorter seq than `o`, we would finish our `for` loop and then need to check if there is some `seq` left over.

Reply:
> In the case where this is a shorter seq than o, we would finish our for loop and then need to check if there is some seq left over.

This is my attempt at adding that check. At this point, we've already called `next_in_place(seq)`, and `seq` will be `nullptr` if there's none left over.

Reply:
We're talking about the same diff, right?

```
-       return true;
+       return seq == nullptr;
```

Reply:
I think I saw the opposite diff. :shrug: My mistake!